### ISSUE TITLE
My  loras are not working in Forge please help

### ISSUE THREAD

**Issue by OhTheHueManatee** [lllyasviel/stable-diffusion-webui-forge#1787]
Everything was working swell earlier but now I'm having issues with my loras when using Forge but not Swarm. When I don't use any Loras the picture is clear. When I use even one the picture is fuzzy especially around the face and eyes . They're also not doing the subject very well sometimes it's totally off like it'll make a man a woman or add a beard to the subject. I'm using Forge and all my settings are default. I've tried updating Forge, updating my GPU and resetting my drivers. In the first picture I don't use any loras. In the other pictures I'm only using one. I'm at a loss as to what else to try. Please help. Thank you.

**Comment by wcole3** [comment]
Two things you might try:
1. Make sure the UI toggle in the top left matches the model type you are trying to run
2. If you are using XL or Flux see if any of the values in the "Diffusion in Low bits dropdown" change behavior

**Comment by OhTheHueManatee** [comment]
Thank you so much for the reply. I have tried both of those things with no luck. I make sure under UI on the left the Flux bubble is filled. I also tried all the option that said "fp16 Lora". It either made blank black pictures or ones that were actually worse. I have also tried deleting and reinstalling Forge but no luck. The only extension I have in Civitai Helper. I tried it without that and still had the same problem. The problem seems to  get worse the more I use Forge. If I close it out and reset it the pictures start out as slightly fuzzy then get much worse with each generation. Again this is only when I use loras.

**Comment by wcole3** [comment]
Is the lora publically available?

**Comment by OhTheHueManatee** [comment]
Yes. I've tried several publicly available loras fron Civitia as well as the ones I've made. None of them are working properly in Forge but working out in Swarm. I just don't like Swarm for any kind of IMG2IMG but Civitai Helper is a God Send.

**Comment by wcole3** [comment]
Can you provide a console log and some settings; I've been unable to reproduce the issue on latest commit with mutliple loras / flux models

**Comment by OhTheHueManatee** [comment]
I will try to do that when I get home later tonight. What kind of settings do you need? Where is the console log?

**Comment by OhTheHueManatee** [comment]
I'm not sure what settings you need so I'll list a few. Let me know if there is anything else you need? 

I'm using the Bruce Campbell Flux Prompt from Civitai.

Prompt: <lora:Bruce_Campbell_FLUX:1.0> color photograph of bruc3campb3ll looking at the viewer of the photo 

UI: Flux Checkpoint: flux1-dev-fp8.safetensors VAE / Text Encoder: AE Clip_l t5xxl_fp16 Swap Location: CPU  Diffusion in Low Bits: Automatic (fp16 Lora)  Sampling method: Euler Schedule type: Simple Sampling steps: 20 Resolution: 1024x1024 High rez fix and refiner unchecked Distilled CFG Scale: 3.5 CFG Scale 1.0

I have tried changing various settings, reinstalling Forge as well a ton of other things with no luck. Like I said it does it with all the loras. 


![00028-2632518669](https://github.com/user-attachments/assets/eaf95fb3-cbe0-4323-a599-7065560e1244)
![00030-648425660](https://github.com/user-attachments/assets/8d353b77-9ab2-4737-95ad-fd19d851415d)
![00031-648425661](https://github.com/user-attachments/assets/8ee60966-7af4-4631-bf77-467a9d18dbb1)
![00031-648425662](https://github.com/user-attachments/assets/2d552080-ae15-4e7d-9e14-bd583f355a41)
![00032-648425662](https://github.com/user-attachments/assets/fc448e88-d5ea-4546-b43d-3eda92ec8618)
![00022-2236742039](https://github.com/user-attachments/assets/2a9f1ce4-9208-413d-876e-b279c1aaafb3)

**Comment by KazmaBlack** [comment]
Hi, did you managed to solve the problem?

**Comment by Tomatoman1900** [comment]
Experience the same and had tried all the options in **Diffusion in Low bits** but didn't work well.
I noticed the runs are bit faster vs auto1111 as mentioned by many, but I think I will just had to settle with auto1111 for now.
ComfyUI is OK, as long as the nodes aren't that many, if running with many nodes I would got headache :)
